benchmark/nixlbench/contrib/Dockerfile

```
@@ -50,21 +56,38 @@ RUN apt-get update -y && \
 # Reinstall needed to fix broken libibverbs-dev, which may lead to lack of Infiniband support.
 # Upgrade is not sufficient if the version is the same since apt skips the installation.
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get -y install \
-    --reinstall libibverbs-dev rdma-core ibverbs-utils libibumad-dev \
-    libnuma-dev librdmacm-dev ibverbs-providers
+RUN if [ "${MAKE_EFA}" != "true" ]; then \
+      DEBIAN_FRONTEND=noninteractive apt-get update && apt-get -y install \
+      --reinstall libibverbs-dev rdma-core ibverbs-utils libibumad-dev \
+      libnuma-dev librdmacm-dev ibverbs-providers; \
+    fi
+
+RUN if [ "${MAKE_EFA}" == "true" ]; then \
```

Hadolint (2.14.0)

[warning] Either use Wget or Curl but not both

(DL4001)